ci/prow/config.yaml

```diff
@@ -359,6 +359,7 @@ presubmits:
       preset-service-account: "true"
       preset-bazel-scratch-dir: "true"
       preset-bazel-remote-cache-enabled: "true"
+      preset-dind-enabled: "true"
```

Comment:
does it have any implication to other repos who don't need this tag?

Reply:
I don't believe so as this is only for `pull-knative-build-integration-tests` job

Reply:
That's correct. It's a setting of a single job.